cmd/ipfs/daemon.go

```diff
@@ -126,16 +126,16 @@ future version, along with this notice. Please move to setting the HTTP Headers.
 	},
 
 	Options: []cmds.Option{
-		cmds.BoolOption(initOptionKwd, "Initialize IPFS with default settings if not already initialized"),
-		cmds.StringOption(routingOptionKwd, "Overrides the routing option (dht, supernode)"),
-		cmds.BoolOption(mountKwd, "Mounts IPFS to the filesystem"),
-		cmds.BoolOption(writableKwd, "Enable writing objects (with POST, PUT and DELETE)"),
-		cmds.StringOption(ipfsMountKwd, "Path to the mountpoint for IPFS (if using --mount)"),
-		cmds.StringOption(ipnsMountKwd, "Path to the mountpoint for IPNS (if using --mount)"),
-		cmds.BoolOption(unrestrictedApiAccessKwd, "Allow API access to unlisted hashes"),
-		cmds.BoolOption(unencryptTransportKwd, "Disable transport encryption (for debugging protocols)"),
-		cmds.BoolOption(enableGCKwd, "Enable automatic periodic repo garbage collection"),
-		cmds.BoolOption(adjustFDLimitKwd, "Check and raise file descriptor limits if needed"),
+		cmds.BoolOption(initOptionKwd, "Initialize IPFS with default settings if not already initialized").Default(false),
+		cmds.StringOption(routingOptionKwd, "Overrides the routing option (dht, supernode)").Default("supernode"),
```

Comment:
it was an old bit of code we wrote to be a 'centralized' routing system. We kindof abandoned it. The code might still work, but its not used.